src/msg/Messenger.h

```diff
@@ -400,6 +400,11 @@ class Messenger {
    */
   virtual int rebind(const set<int>& avoid_ports) { return -EOPNOTSUPP; }
   /**
+   * @return 1 on unlinked, or 0 on linked, or -errno if
+   * we can be more specific about the failure.
+   */
+  virtual int is_linked() { return -EOPNOTSUPP; }
```

Comment:
1 = unlinked
0 = linked
seems to me to be with opposition with code for Accepter::is_linked()